compiler-rt/lib/scudo/standalone/mem_map_linux.cpp

```diff
@@ -129,7 +129,7 @@ void MemMapLinux::releaseAndZeroPagesToOSImpl(uptr From, uptr Size) {
 bool ReservedMemoryLinux::createImpl(uptr Addr, uptr Size, const char *Name,
                                      uptr Flags) {
   ReservedMemoryLinux::MemMapT MemMap;
-  if (!MemMap.map(Addr, Size, Name, Flags | MAP_NOACCESS))
```

Comment:
I think the choice of whether to pass `MAP_ALLOWNOMEM` or not can be left to the caller, like we do in `ReservedMemoryFuchsia::createImpl`

Reply:
As mentioned in the description, this is just a proof-of-concept so most stuff are quick changes. The formal CL will review all the logic later